protocol/avdl/keybase1/teams.avdl

```diff
@@ -619,26 +618,41 @@ protocol teams {
   @typedef("string")
   record TeamInviteName {}
 
+  // For Keybase, Seitan, Invitelink, etc. invites, we want to show a different
+  // invite name than what is in the sigchain (which may be an encoded
+  // userversion or a messagepack blob, e.g.)
+  @typedef("string")
+  record TeamInviteDisplayName {}
+
   record TeamInvite {
     TeamRole role;
     TeamInviteID id;
     TeamInviteType type;
+
+    // For SBS, name is the username (`type` will specify the service)
+    // for phone/email, name is the number/address
+    // for keybase-type invite, name is the percent-form UV
+    // for seitan or invitelink, name is the messagepack encoded pkey
     TeamInviteName name;
+
     UserVersion inviter;
     union { null, TeamInviteMaxUses } maxUses;
     union { null, UnixTime } etime;
   }
 
   record AnnotatedTeamInvite {
-    TeamRole role;
-    TeamInviteID id;
-    TeamInviteType type;
-    TeamInviteName name;
-    UserVersion uv;
-    UserVersion inviter;
+    TeamInvite invite;
```

Comment:
Embedded TeamInvite here so we don't accidentally drop new fields if we add them to TeamInvite